src/Microsoft.Data/DataFrameColumnArithmeticTemplate.cs

```diff
@@ -0,0 +1 @@
+﻿
```

Comment:
Does templating require the empty file be in the repo for some reason?

Reply:
Removed. I had to rebase on upstream/master and it got generated this time.